app/routes/session.py

```diff
@@ -150,3 +172,28 @@ def get_sign_out():
 @session_blueprint.route("/signed-out", methods=["GET"])
 def get_signed_out():
     return render_template(template="signed-out")
+
+
+def get_runner_claims(decrypted_token):
+    try:
+        return (
+            validate_runner_claims_v2(decrypted_token)
+            if decrypted_token.get("version") == Version.V2.value
+            else validate_runner_claims(decrypted_token)
+        )
```

Comment:
Check this explicitly otherwise invalid values fall back to v1.
```py
if version:
	if v2:
		return v2_payload()
	
	raise error

return v1_payload()

Reply:
Updated this 👍

Reply:
Outstanding but left another comment, so resolving this.